derive_builder/CHANGELOG.md

```diff
@@ -1,261 +1,309 @@
 # Change Log
+
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [0.12.1] 2023-04-06
+
+-   Inherit `strip_option` configuration from `setter` for `try_setter`, avoiding compilation errors #284
```

Comment:
```suggestion
- Make try-setters inherit `strip_option` from `setter` for `try_setter`. Using these settings together previously caused a compile error  #284
```

The use of "inherit... from... for" is slightly awkward.

Reply:
Updated the message to reflect this after rebasing.